code-env/python/spec/requirements.txt

```diff
@@ -1 +1,14 @@
+spacy==3.0.0
+emoji==0.6.0
```

Comment:
You need to replace that with spacymoji (which is required by the multilingual tokenizer module)
spacymoji takes care of installing the right version of `emoji`

Reply:
Problem is that spacymoji is not compatible with spacy 3.0 😕 
So you will need to remove the spacymoji from the tokenizer module.